src/tallies/tally_scoring.cpp

```diff
@@ -1148,6 +1149,139 @@ score_general_ce(const Particle* p, int i_tally, int start_index,
       break;
 
 
+    case SCORE_HEATING:
+      if (p->type_ == Particle::Type::neutron) {
+        score_bin = NEUTRON_HEATING;
```

Comment:
Just a small request here: I'd prefer not to update the `score_bin` variable and instead just use `reaction_index_[NEUTRON_HEATING]` below.  It's a little confusing to change the variable inspected by the `switch` statement inside one of the cases, and the compiler might even add some optimization by recognizing that `score_bin` is const